Scripts/Python/kdshGlowInTheDark.py

```diff
@@ -295,14 +271,14 @@ def OnNotify(self,state,id,events):
 
                 if TimerID==1:
                     PtDebugPrint("kdshGlowInTheDark: Timer 1 Callback. Raising elevator again.")
-                    xRgnBottom.clearNow(self.key)
                     rgnExitSubBtm.disable()
 
                     if not self.sceneobject.isLocallyOwned():
                         PtDebugPrint("\tI'm not the owner, so I'll let another client netforce raise the elevator.")
                         return
                     else:
                         respElevUp.run(self.key)
+                        PtAtTimeCallback(self.key,14,6)
```

Comment:
Last thing, I think. Since this timer sends a local notification that toggles an exclude region, it needs to be sent without regard to the ownership of the PFM object. In other words, move this up to where `xRgnButton.clearNow(self.key)` was. Otherwise, there will be some sneaky bugs around players quitting the game or linking out while the elevator is moving.

Reply:
Change made.